src/main/java/org/spongepowered/api/block/BlockSnapshot.java

```diff
@@ -201,7 +204,19 @@ static Builder builder() {
          * @return This builder, for chaining
          */
         Builder notifier(UUID uuid);
+    }
 
+    /**
+     * Represents a factory to create {@link BlockSnapshot}s.
+     */
+    interface Factory {
+
+        /**
+         * Represents a {@link BlockSnapshot} with the default state of
+         * {@link BlockTypes#AIR} and a {@link ServerLocation} that cannot be determined.
```

Comment:
Not sure "cannot be determined" is the appropriate phrasing.

Maybe state that the location is up to the implementation as to where "default" snapshots occur at.

As an example, In the mcp implementation, it would be 0, 0, 0 but Lantern could put it at Integer.MIN_VALUE.

Reply:
Fixed